sim/paladin/items.go

```diff
@@ -34,4 +38,76 @@ func init() {
 			character.PseudoStats.MobTypeAttackPower += 36
 		}
 	})
+
+	core.NewItemEffect(HerosBrand, func(agent core.Agent) {
+		//Increases critical strike chance of holy shock spell by 2%
+		paladin := agent.GetCharacter()
+		paladin.OnSpellRegistered(func(spell *core.Spell) {
+			if spell.SpellCode == SpellCode_PaladinHolyShock {
+				spell.BonusCritRating += 2.0
+			}
+		})
+	})
+
+	
+	core.NewItemEffect(zandalarFreethinkersBreastplate, func(agent core.Agent) {
+		character := agent.GetCharacter()		
+		character.PseudoStats.SchoolBonusHitChance[stats.SchoolIndexHoly] += 3
+	})
+
+	
+	core.NewItemEffect(zandalarFreethinkersBelt, func(agent core.Agent) {
+		character := agent.GetCharacter()		
+		character.PseudoStats.SchoolBonusHitChance[stats.SchoolIndexHoly] += 2
+	})
+
+
+	// https://www.wowhead.com/classic/item=231285/grileks-charm-of-valor
+	// Use: Increases the critical hit chance of Holy spells by 10% for 15 sec. If Shock and Awe is engraved, gain an additional 5%. (1 Min, 30 Sec Cooldown)
+	core.NewItemEffect(grileksCharmOfValor, func(agent core.Agent) {
+		character := agent.GetCharacter()
+		paladin := agent.(PaladinAgent).GetPaladin()
+
+		character.PseudoStats.BonusDamage += 4
```

Comment:
This might be a copy-paste error? I don't see a bonus damage effect on this trinket